### PR TITLE
Add sphinx configuration

### PR DESCRIPTION
**Pull request by dweindl**
Add some basic sphinx configuration and make docstrings sphinx-compatible.

`cd doc && make html && xdg-open _build/html/index.html`

Related to #7

### CODE REVIEW

File: petab_select/constants.py [hunk 1]
```diff
@@ -106,7 +106,7 @@
 # Parameters can be fixed to a value, or estimated if indicated with the string
 # `ESTIMATE`.
 # TODO change to `Literal[ESTIMATE]` (Python >= 3.8)
```
Comment (dweindl) [inline petab_select/constants.py:108]:
```suggestion
```

**Comment by dweindl** [comment]
@dilpath : This allows generating basic API documentation. More fixes and readthedocs configuration in #63. You decide whether you want to merge this, or prettify the documentation first. Or whether I should split it into separate PRs for sphinx-config and docstrings.

File: petab_select/problem.py [hunk 2]
```diff
@@ -29,12 +29,13 @@ class Problem(abc.ABC):
             The model space.
         calibrated_models:
             Calibrated models. Will be used to augment the model selection problem (e.g.
-            by excluding them from the model space). FIXME refactor out
+            by excluding them from the model space).
+            FIXME refactor out
```
Comment (dilpath) [inline petab_select/problem.py:33]:
```suggestion
            FIXME(dilpath) refactor out
```

**Review by dilpath — APPROVED** [review]
Nice! Thanks

**Comment by dilpath** [comment]
> @dilpath : This allows generating basic API documentation. More fixes and readthedocs configuration in #63. You decide whether you want to merge this, or prettify the documentation first. Or whether I should split it into separate PRs for sphinx-config and docstrings.

Sounds good :+1: I'll merge this already since it passes the non-RTD tests.

**Pull request merged**